lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
Yes, the idea is that this will capture the *default* formatting for the various languages. Anything custom is the user's responsibility, until we allow configuring the parsing regexps.